frontend/app/mocks/node.ts

```diff
@@ -1,21 +1,55 @@
+import jsonpatch, { JsonPatchError, type Operation } from 'fast-json-patch';
 import { HttpResponse, http } from 'msw';
 import { setupServer } from 'msw/node';
+import { z } from 'zod';
+
+import { db } from '~/mocks/db';
+
+const userIdSchema = z.string().uuid();
 
 const handlers = [
-  // prettier-ignore
-  http.get('https://api.example.com/users/*', () => {
+  http.get('https://api.example.com/users/:id', ({ params }) => {
+    const parsedUserId = userIdSchema.safeParse(params.id);
+    const userEntity = parsedUserId.success ? db.user.findFirst({ where: { id: { equals: parsedUserId.data } } }) : undefined;
```

Comment:
Thoughts on formatting this better? It's hard to read as-is (in my opinion).

Reply:
Fixed